client/src/components/NodeLocations.js

```diff
@@ -1,50 +1,63 @@
 import React from 'react'
 import { withStyles } from '@material-ui/core/styles'
 import Grid from '@material-ui/core/Grid'
+import { Typography } from '@material-ui/core'
+
+import { COLOR_DARKEST_GREY, COLOR_MEDIUM_GREY } from '../utils/theme'
 import CustomMap from '../components/CustomMap'
 import CustomHeader from '../components/CustomHeader'
 import LocationRow from '../components/LocationRow'
 import CustomProgress from '../components/CustomProgress'
 import Anchor from '../components/Anchor'
-import { Typography } from '@material-ui/core';
 
 const styles = theme => ({
   root: {
     marginBottom: theme.spacing.unit * 10,
   },
-  locationInfo: {
-    boxShadow: '0px -30px 0px -29px black, 0px 30px 0px -29px black',
+  column: {
+    boxShadow: `0px -30px 0px -29px ${COLOR_MEDIUM_GREY}, 0px 30px 0px -29px ${COLOR_MEDIUM_GREY}`,
   },
 })
 
 class NodeLocations extends React.Component {
+  renderList = () => {
+    const { nodesPerCountry } = this.props
+    const countriesSorted = Object.keys(nodesPerCountry).sort(function(a, b) {
+      return nodesPerCountry[b] - nodesPerCountry[a]
+    })
+    const numOfNodesSorted = countriesSorted.map(key => nodesPerCountry[key])
+    return countriesSorted.map((country, i) => {
+      if (i < 9) {
```

Comment:
could use a slice on the `countriesSorted` array instead of this conditional expression